src/app/reporting/Engine.cpp

```diff
@@ -489,10 +488,35 @@ void Engine::Run()
 
     if (allReadClean)
     {
-        InteractionModelEngine::GetInstance()->ReleaseClusterInfoList(mpGlobalDirtySet);
+        mGlobalDirtySet.ReleaseAll();
     }
 }
 
+bool Engine::MergeOverlappedAttributePath(ClusterInfo & aAttributePath)
+{
+    bool overlapped = false;
+    mGlobalDirtySet.ForEachActiveObject([&](auto * path) {
+        // If overlapped, we would skip this target path,
+        // --If targetPath is part of previous path, return true
+        // --If previous path is part of target path, update filedid and listindex and mflags with target path, return true
```

Comment:
What is "filedid"?  What is "mflags"?

Reply:
that is from previous code, updated, thanks